integration_tests/e2e/induction/createChangeInductionQuestionSet.cy.ts

```diff
@@ -94,9 +93,23 @@ context(`Change new Induction question set by updating 'Hoping to work on releas
       .hasBackLinkTo(`/prisoners/${prisonNumberForPrisonerWithNoInduction}/create-induction/qualifications`)
       .submitPage()
 
+    // 'Has the prisoner worked before' is the next page. This is asked on the long question set, so this will already have an answer set
```

Comment:
Add the new page interactions to the cypress test that tests "creating a Long question set induction; but change it to a short question set induction via the change link for 'do they want to work on release' on the Check Your Answers page